cvat-ui/src/cvat-logger.ts

```diff
@@ -1,12 +1,59 @@
 // Copyright (C) 2020-2022 Intel Corporation
+// Copyright (C) CVAT.ai Corporation
 //
 // SPDX-License-Identifier: MIT
 
+import ErrorStackParser from 'error-stack-parser';
 import { getCore } from 'cvat-core-wrapper';
+import { getCVATStore } from 'cvat-store';
+import { CombinedState } from 'reducers';
 
 const core = getCore();
 const { logger } = core;
 const { EventScope } = core.enums;
 
+export function logError(error: unknown, save: boolean, extras = {}): void {
```

Comment:
It seems that `type` is now always specified in `extras`, so we could consider adding type definitions for this parameter.
Or maybe add a default `type: error` if its not specified